src/WarcraftLegacies.Source/Quests/Lordaeron/QuestKingArthas.cs

```diff
@@ -38,11 +42,11 @@ public sealed class QuestKingArthas : QuestData
 
     /// <inheritdoc/>
     protected override string RewardFlavour =>
-      "With the Lich King eliminated, the Kingdom of Lordaeron is free of its greatest threat. King Terenas Menethil proudly abdicates in favor of his son.";
+      "With a buffer outpost in Northrend, the Kingdom of Lordaeron is safer of its greatest threat. King Terenas Menethil proudly abdicates in favor of his son, now an experienced warrior.";
 
     /// <inheritdoc/>
     protected override string RewardDescription =>
-      "Arthas becomes the King of Lordaeron, gains the Crown of Lordaeron, and he can no longer permanently die";
+      $"Arthas becomes the King of Lordaeron, gains the Crown of Lordaeron, and he can no longer permanently die. You can now build {GetObjectName(Constants.UNIT_H06C_HIGH_TOWER_LORDAERON_SPECIALIST)}. Your Knights and Huntsman will also be upgraded to stronger units.";
```

Comment:
```suggestion
      $"Arthas becomes the King of Lordaeron, gains the Crown of Lordaeron, and he can no longer permanently die. Learn to build {GetObjectName(Constants.UNIT_H06C_HIGH_TOWER_LORDAERON_SPECIALIST)}s. Your Knights and Huntsman are upgraded to stronger units";
```
I think you also want to say like, "Knights and Humans are upgraded to x and y". This is a bit vague right now.

Reply:
i was afraid the text would be too long

Reply:
That window has a scroll bar so it's not really a concern. If the player doesn't scroll, they get the same information they do now. If they do, they get all the info.

Reply:
ok!